cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -127,25 +127,28 @@ public TableRenderer getTableRenderer() {
   }
 
   public Id.Namespace getCurrentNamespace() {
-    return clientConfig.getNamespace();
+    return connectionConfig != null ? connectionConfig.getNamespace() : null;
```

Comment:
And it seems like a lot of the usages in the CLI use this method without expecting it to be able to return null.